src/components/rules/rules.ts

```diff
@@ -0,0 +1,71 @@
+import './rules.scss';
+import WebComponent, { createElementFromString } from '../../common/WebComponent';
+
+class RulesComponent implements WebComponent {
+  render(): Element {
+    return createElementFromString(`<main class="box">
```

Comment:
<main> tag is used to specify the main content of the <body>. What if I would like to use this component, but also the other one or if I would like to use this component twice?